process/block/preprocess/scheduledTxsExecution.go

```diff
@@ -196,6 +211,12 @@ func getAllIntermediateTxsAfterScheduledExecution(
 
 // GetScheduledSCRs gets the resulted SCRs after the execution of scheduled transactions
 func (ste *scheduledTxsExecution) GetScheduledSCRs() map[block.Type][]data.TransactionHandler {
+	mapScheduledSCRs, _ := ste.createCopyScheduledSCRs()
+
+	return mapScheduledSCRs
+}
+
+func (ste *scheduledTxsExecution) createCopyScheduledSCRs() (map[block.Type][]data.TransactionHandler, int) {
```

Comment:
This method is used only in GetScheduledSCRs method above which has only two lines of code. I think that this code could be merged there and also for the debugging scope we could keep the logs as they were before.

Reply:
yes, previously I was using it in two places, removed, and added back the logs